### ISSUE TITLE
Show picture of caliper

### ISSUE THREAD

**Issue by probonopd** [duckythescientist/caliperRead#1]
This sketch would be much more useful if we would know which caliper to order. Is it the same as in https://sites.google.com/site/marthalprojects/home/arduino/arduino-reads-digital-caliper?

**Comment by duckythescientist** [comment]
Agreed. I'll get to it shortly. And yes, those are the same calipers, but their code has no timeouts. Infinite loops on embedded systems without timeouts are scary/dangerous. 

The calipers for this project are the cheap $15ish ones on places such as Amazon. You get what you pay for though, but I can't justify the super nice calipers. 

I also have some board designs for this project, but I haven't given the project any TLC recently.
